src/blocks/utils/SeamMeme/utils/getMemes.ts

```diff
@@ -0,0 +1,14 @@
+import axios from "axios"
+import type { Meme } from "../types/types"
+
+const getMemes = async (): Promise<Meme[]> => {
```

Comment:
this can live directly in the Composer component, to avoid having to create another file + to colocate the functionality where it is used. prefer colocation when the function isn't used in multiple places to increase code readability.